dashboard/src/components/Filters.tsx

```diff
@@ -116,20 +124,31 @@ export const filterItem =
     return item;
   };
 
-export const filterData = (data, filters) => {
-  console.log({ filters });
+export const filterData = (data: any[], filters: Array<Filter>) => {
   data = data.map(filterItem(filters)).filter(Boolean);
   return data;
 };
 
-const Filters = ({ onChange, base, filters, title = "Filtres :", saveInURLParams = false }) => {
+const Filters = ({
+  onChange,
+  base,
+  filters,
+  title = "Filtres :",
+  saveInURLParams = false,
+}: {
+  onChange: (filters: Array<Filter>, saveInURLParams: boolean) => void;
+  base: Array<FilterableField>;
+  filters: Array<Filter>;
```

Comment:
Personnellement je trouve plus lisible le `[]` mais c'est une affaire de gout. On a au moins trois choix : 
 - On fait tous les deux `unknown[]`
 - On fait tous les deux `Array<unknown>`
 - On s'en fout on mix

Les trois me vont :D (la 2 un peu moins, mais ça va !)

```suggestion
  onChange: (filters: Filter[], saveInURLParams: boolean) => void;
  base: FilterableField[];
  filters: Filter[];
```

Reply:
je mets des `Array` parce que cet article m'avait "convaincu", non pas que j'eusse un avis important sur la chose puisque je ne faisais pas beaucoup de TS: https://tkdodo.eu/blog/array-types-in-type-script

donc bon, j'ai pas d'opinion forte, et je voterai plutôt la 3 !

mais si tu as une opinion forte pour la 1, no problemo